src/utils/zendesk.ts

```diff
@@ -1,13 +1,42 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { ZENDESK_URL_PREFIXES } from "settings";
 import { loadScript } from "./loadScript";
 
 const ZENDESK_SCRIPT_SRC =
   "https://static.zdassets.com/ekr/snippet.js?key=57ad1fbc-d174-4561-8102-b3c8d98436d5";
```

Comment:
We should extract this into env variables + map through as per https://github.com/sparkletown/internal-sparkle-issues/issues/100